requirements.txt

```diff
@@ -0,0 +1 @@
+pre-commit==1.14.0
```

Comment:
As I mentioned about using [requirements-tools](https://github.com/Yelp/requirements-tools) (which `update-requirements` is a part of), I believe you'd want to rename this to `requirements-dev.txt` and then have just `pre-commit` by itself unpinned in a `requirements-dev-minimal.txt` so it can be upgraded from just that base version.